rank/data/tests/images.json

```diff
@@ -0,0 +1,142 @@
+[
+    {
+        "id": "precision",
+        "description": "TBD",
+        "eval": "equalTo1",
+        "cases": [
+            {
+                "searchTerms": "crick dna sketch",
+                "ratings": [
+                    "gzv2hhgy"
+                ]
```

Comment:
Another "not for now" problem: I wonder if we could change this data structure slightly to allow a comment alongside these IDs?

I have to look them up in /works to see what they are, which is fine right now, but could cause issues if e.g. IDs get suppressed or redirected – a brief comment about the intention might be helpful.

```
"ratings": [
    {
        "id": "gzv2hhgy",
        "comment": "Pencil sketch of the DNA double helix by Francis Crick"
    }
]
```